src/error.rs

```diff
@@ -1,26 +1,30 @@
 use std::{error, fmt};
+use regex;
```

Comment:
Since we are just referring to the regex package's error type, can we omit the `use` statement and just refer to it as `::regex::Error`?